gtsfm/configs/deep_front_end_astronet.yaml

```diff
@@ -0,0 +1,63 @@
+SceneOptimizer:
+  _target_: gtsfm.scene_optimizer.SceneOptimizer
+  save_gtsfm_data: True
+  save_two_view_correspondences_viz: False
+  save_3d_viz: True
+  pose_angular_error_thresh: 5 # degrees
+
+  feature_extractor:
+    _target_: gtsfm.feature_extractor.FeatureExtractor
+    detector_descriptor:
+      _target_: gtsfm.frontend.cacher.detector_descriptor_cacher.DetectorDescriptorCacher
+      detector_descriptor_obj:
+        _target_: gtsfm.frontend.detector_descriptor.superpoint.SuperPointDetectorDescriptor
+        max_keypoints: 5000
+
+  two_view_estimator:
+    _target_: gtsfm.two_view_estimator.TwoViewEstimator
+    bundle_adjust_2view: True
+    eval_threshold_px: 4 # in px
+    bundle_adjust_2view_maxiters: 100
+
+    matcher:
+      _target_: gtsfm.frontend.cacher.matcher_cacher.MatcherCacher
+      matcher_obj:
+        _target_: gtsfm.frontend.matcher.superglue_matcher.SuperGlueMatcher
+        use_outdoor_model: True
+
+    verifier:
+      _target_: gtsfm.frontend.verifier.ransac.Ransac
+      use_intrinsics_in_verification: True
+      estimation_threshold_px: 4 # for H/E/F estimators
+
+    inlier_support_processor:
+      _target_: gtsfm.two_view_estimator.InlierSupportProcessor
+      min_num_inliers_est_model: 15
+      min_inlier_ratio_est_model: 0.1
+
+  multiview_optimizer:
+    _target_: gtsfm.multi_view_optimizer.MultiViewOptimizer
+    rot_avg_module:
+      _target_: gtsfm.averaging.rotation.shonan.ShonanRotationAveraging
+
+    trans_avg_module:
+      _target_: gtsfm.averaging.translation.averaging_1dsfm.TranslationAveraging1DSFM
+      robust_measurement_noise: True
+
+    data_association_module:
+      _target_: gtsfm.data_association.data_assoc.DataAssociation
+      triangulation_options:
+        _target_: gtsfm.data_association.point3d_initializer.TriangulationOptions
+        reproj_error_threshold: 100
+        min_track_len: 2
+        mode:
+          _target_: gtsfm.data_association.point3d_initializer.TriangulationParam
+          value: 1 # 0 corresponds to NO_RANSAC
+        max_num_hypotheses: 100
+      save_track_patches_viz: False
```

Comment:
@travisdriver does the RANSAC triangulation setting hurt the other benchmarks (besides AstroNet)? If so, do we know why?

Reply:
The RANSAC triangulation doesn't seem to be hurting any of the relevant benchmarks: see [this comment](https://github.com/borglab/gtsfm/pull/292#issuecomment-999275501). 

It significantly improves AstroNet with a more appropriate threshold of 100 pixels (since the averaged poses are still not very good). It's all a function of the accuracy of the averaged poses.